less/entities.less

```diff
@@ -360,3 +360,46 @@
 		}
 	}
 }
+
+/* premium class card filter colors, for use with red overlay image */
+
+.druid-color {
+	filter: hue-rotate(135deg) saturate(40%) brightness(95%);
+}
+
+.hunter-color {
+	filter: hue-rotate(150deg) saturate(80%) brightness(100%);
+}
+
+.mage-color {
+	filter: hue-rotate(303deg) saturate(88%) brightness(94%);
+}
+
+.paladin-color {
+	filter: hue-rotate(20deg) saturate(90%) brightness(90%);
+}
+
+.priest-color {
+	visibility: hidden;
+}
+
+.rogue-color {
+	filter: hue-rotate(104deg) saturate(0%) brightness(80%);
+}
+
+.shaman-color {
+	filter: hue-rotate(250deg) saturate(75%) brightness(80%);
```

Comment:
Color looks wrong, looks right for me with about `filter: hue-rotate(220deg) saturate(50%) brightness(100%);`